packages/nx/src/nx-init/react/index.ts

```diff
@@ -126,7 +102,6 @@ async function normalizeOptions(
     npxYesFlagNeeded,
     isVite,
     isStandalone,
-    integrated,
```

Comment:
there is code in this file that uses `integrated` option:
```
  const buildCommand = options.integrated
    ? `npx nx build ${options.reactAppName}`
    : 'npm run build';
```
does the code still run with the integrated option removed?